Docs/source/building/juwels.rst

```diff
@@ -0,0 +1,67 @@
+.. _building-juwels:
+
+Juwels (JSC)
+============
+
+The `Juwels supercomputer <https://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JUWELS/JUWELS_node.html>`_ is located at JSC.
+
+See `this page <https://apps.fz-juelich.de/jsc/hps/juwels/quickintro.html>`__ for a quick introduction.
```

Comment:
Isn't there also a good amount of HPC filesystems mounted, maybe we can hint them here as on Summit.
```suggestion
See `this page <https://apps.fz-juelich.de/jsc/hps/juwels/quickintro.html>`_ for a quick introduction.

* Batch system: `Slurm <https://apps.fz-juelich.de/jsc/hps/juwels/quickintro.html#batch-system-on-system-name>`_
* `Production directories <https://apps.fz-juelich.de/jsc/hps/juwels/environment.html?highlight=scratch#available-filesystems>`_:
  * ``$SCRATCH``: Scratch filesystem for temporary data (90 day purge)
  * ``$FASTDATA/``: Storage location for large data (backuped)
  * Note that the ``$HOME`` directory is usually very slow and running there will impact performance.
```

Reply:
Thanks for adding these. Unfortunately, by default, I think neither of `$SCRATCH` and `$FASTDATA` are defined, or at least not in my experience. There is a `$SCRATCH_<project ID>` and a shared `$PROJECT_<project ID>` though. I will ping the Juwels support to understand the status of this documentation page.